Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml

```diff
@@ -3,7 +3,7 @@ commonfields:
   id: WildFire-v2
   version: -1
 configuration:
-- display: Server URL (e.g. https://192.168.0.1)
+- display: Server base URL (e.g. https://192.168.0.1/publicapi)
```

Comment:
why publicapi is not part of the base url in the code?

Reply:
Because there are users that have already configured it themselves.
If you think it should be changed I can check in the code if they added it and if not then add it, but I wasn't sure if it's necessary

Reply:
you can add a simple code check to make sure that it is there, and if not add a corresponding error msg or just add that yourself

Reply:
adding an error msg in test-module or something is not such a good idea if we want to allow both options (with/without the suffix), so I just add the suffix if it's not there